test/interpreter/function-text.spec.ts

```diff
@@ -160,18 +160,24 @@ describe('time duration', () => {
   it('works', () => {
     const engine = HyperFormula.buildFromArray([
       ['0.1', '=TEXT(A1, "[hh]:mm:ss")'],
-      ['1.1', '=TEXT(A2, "[hh]:mm:ss")', ],
+      ['1.1', '=TEXT(A2, "[hh]:m:ss")', ],
```

Comment:
If `[hh]:m:ss` and `[hh]:mm:ss` are currently equivalent, then I think it would be good to keep the old test besides adding a new one, to emphasize that there is no breaking behavior.

Reply:
done